src/main/java/com/adit/backend/domain/auth/service/CustomUserService.java

```diff
@@ -0,0 +1,111 @@
+package com.adit.backend.domain.auth.service;
+
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.oauth2.client.userinfo.DefaultOAuth2UserService;
+import org.springframework.security.oauth2.client.userinfo.OAuth2UserRequest;
+import org.springframework.security.oauth2.core.OAuth2AccessToken;
+import org.springframework.security.oauth2.core.OAuth2AuthenticationException;
+import org.springframework.security.oauth2.core.user.DefaultOAuth2User;
+import org.springframework.security.oauth2.core.user.OAuth2User;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+@Service
+public class CustomUserService extends DefaultOAuth2UserService {
+
+	private static final String NaverRegistrationId = "naver";
```

Comment:
상수 네이밍은 대문자 + 문자 구분 간 "_"을 이용합니다!
```suggestion
	private static final String NAVER_REGISTRATION_ID = "naver";
```